.env

```diff
@@ -11,6 +11,8 @@ MAIL_SENDER = affigeninc@gmail.com
 # MAIL_RECEIVERS = tunisia@gentaur.com info@affigen.com
```

Comment:
This repository is no longer supported please start using GitLab in the future thank you